### PR TITLE
update_value func moved to dictutils.h

### PR DESCRIPTION
**Pull request by ashvish183**
Merge it or tell my where I am wrong

### CODE REVIEW

File: sli/dictutils.h [hunk 1]
```diff
@@ -251,6 +251,36 @@ updateValue( DictionaryDatum const& d, Name const n, VT& value )
   value = getValue< FT >( t );
   return true;
 }
+/* Polymorphic version of update_value.
```
Comment (heplesser) [inline sli/dictutils.h:254]: Adjust this comment to the same (doxygen) style as the other function-describing comments in `dictutils.h`.

**Review by heplesser — CHANGES_REQUESTED** [review]
@ashvish183 Thank you for your contribution. Could you contact me by email (info@nest-initiative.org) about some formalities?

File: sli/dictutils.h [hunk 2]
```diff
@@ -251,6 +251,36 @@ updateValue( DictionaryDatum const& d, Name const n, VT& value )
...
+ * This code will take either an int or a double and convert it to an
+ * int.
+ */
+bool
```
Comment (stinebuu) [inline sli/dictutils.h:258]: You need an `inline` before `bool` to make the code compile. This is why the Github actions fail. So, it needs to be `inline bool` here.

**Comment by heplesser** [comment]
@ashvish183 Do you plan to work more on this PR? Otherwise, I will close it due to inactivity.

**Comment by jougs** [comment]
Closing due to inactivity.

**Comment by jougs** [comment]
@stinebuu: please feel free to adopt the underlying branch and create a new PR from it. I think copyright is not an issue here, as this simply moves already existing code from one place to another.

**Pull request closed without merging**